packages/framework-types/src/config.ts

```diff
@@ -225,12 +191,6 @@ export class BoosterConfig {
   }
 }
 
-export const JWT_ENV_VARS = {
```

Comment:
@javiertoledo We were keeping this env variables to be backwards compatible with the aws cognito rocket, [take a look at this](https://github.com/boostercloud/rocket-auth-aws-infrastructure/blob/809907db7bc32eb583210832eec60b62a3ced434/src/auth-stack.ts#L398). Anyway, I'm totally agree of removing this but probably some work will be needed in the aws cognito rocket to use the future Booster versions.